doc/guides/contributing/pull-requests.md

```diff
@@ -308,15 +308,8 @@ $ git push --force-with-lease origin my-branch
 **Important:** The `git push --force-with-lease` command is one of the few ways
 to delete history in `git`. Before you use it, make sure you understand the
 risks. If in doubt, you can always ask for guidance in the pull request.
```

Comment:
@Farenheith thanks! I think it would make more sense to put it before `Before you use it`, so all the drawbacks are listed in a row. Also I realize now that contributors may not know what "force-pushing" is, maybe it's best to keep it as:
```suggestion
to delete history in `git`. It also complicates the review process, as it won't
allow reviewers to get a quick glance on what changed. Before you use it, make
sure you understand the risks. If in doubt, you can always ask for guidance in
the pull request.
```

Reply:
@aduh95 that makes sense. I updated the PR